Donation-platform/src/main/java/com/google/CharityOld.java

```diff
@@ -21,7 +21,7 @@
  * Class to encapsulate the attributes of a charity loaded from the database
  * and to send information to the database regarding a charity
  */
-public final class Charity {
+public final class CharityOld {
```

Comment:
what about this old? Update documentation accordingly

Reply:
we had two versions of Charity.java initially, and now we have merged into one, but I don't want to delete the old class until I can confirm that everything works with the new one. Will update the comments to reflect this.

Reply:
sg